src/features/people/PeopleDetails/peopleDetailsSaga.js

```diff
@@ -0,0 +1,24 @@
+import { call, put, takeLatest, select } from "redux-saga/effects";
+import {
+    fetchDataSuccess,
+    selectPersonId,
+    fetchDataError,
+    getDetailsForPerson,
+} from "./peopleDetailsSlice";
+import { getPeopleDetails } from "./getPeopleDetails";
+
+function* fetchPersonDetailsHandler() {
+    try {
+        console.log("Saga: Handling getDetailsForPerson action");
```

Comment:
Do we need that log in deployed app?

Reply:
Thank you for catching that! 😀The console log was only for debugging purposes during the coding phase, and I forgot to remove it. It has no purpose in the deployed app, so I have now removed the unnecessary logs 😁
Appreciate your sharp eye for details! 😀